framework/log/DbTarget.php

```diff
@@ -80,13 +84,16 @@ public function export()
                     $text = VarDumper::export($text);
                 }
             }
-            $command->bindValues([
-                ':level' => $level,
-                ':category' => $category,
-                ':log_time' => $timestamp,
-                ':prefix' => $this->getMessagePrefix($message),
-                ':message' => $text,
-            ])->execute();
+            if ($command->bindValues([
+                    ':level' => $level,
+                    ':category' => $category,
+                    ':log_time' => $timestamp,
+                    ':prefix' => $this->getMessagePrefix($message),
+                    ':message' => $text,
+                ])->execute() > 0) {
+                continue;
+            }
+            throw new InvalidValueException('Unable to export log through database!');
```

Comment:
I agree. I used this one only because all other exceptions seem to be inappropriate and I did not want to use general `yii\base\Exception`. Should I change it to `yii\base\Exception`?